src/Equations/PartialScreeningFactorWalkowiak.cpp

```diff
@@ -0,0 +1,72 @@
+#include<iostream>
```

Comment:
I guess that these functions are described in a paper somewhere (perhaps https://doi.org/10.1063/5.0075859 ?) Maybe you could add a comment at the top of this which just briefly describes what is done and refers to the relevant paper?

Reply:
Done